Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -229,34 +232,45 @@ namespace AZ::Render
         EMotionFX::JointSelectionRequestBus::BroadcastResult(
             cachedSelectedJointIndices, &EMotionFX::JointSelectionRequests::FindSelectedJointIndices, instance);
 
+        size_t cachedHoveredJointIndex;
```

Comment:
update: We should initialize `cachedHoveredJointIndex` to `0`

Reply:
Apologies this should more likely be `InvalidIndex`, but we should initialize it to something 😅

Reply:
oh yeah, good call on initializing it